deploy-helm/helm-operator-deployment.yaml

```diff
@@ -14,6 +14,10 @@ spec:
     spec:
       serviceAccount: flux
       volumes:
+      - name: sshdir
+        configMap:
+          name: {{ template "flux.fullname" . }}-ssh-config
```

Comment:
This one isn't in a template. In the flux deployment YAML, I've commented it and the mount out, with an explanation, since they require the configmap to exist.